gatsby-node.js

```diff
@@ -192,18 +105,14 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
     })
   })
 
-  tutsInfo.forEach((tutInfo, ind) => {
-    const res = tutsGqlResult[`${ind}`].value.data.res.nodes
-    res.forEach(({ childMdx: node }) => {
-      createPage({
-        path: `${node.frontmatter.slug}/`,
-        component: tutorialTemplate,
-        context: {
-          slug: `${node.frontmatter.slug}`,
-          version: tutInfo.version,
-          navMenuSlug: tutInfo.navSlug,
-        },
-      })
+  allTuts.forEach(({ node }) => {
+    createPage({
+      path: `${node.frontmatter.slug}/`,
+      component: tutorialTemplate,
+      context: {
+        slug: `${node.frontmatter.slug}`,
+        version: `3.0`,
```

Comment:
this is a bug - versions should be per tutorial... A refactor may be needed to allow for this?

Reply:
@NukeManDan Yes.  When I worked on this during the retreat, it wasn't clear how to handle the versioning.  I will need to work on another PR to make sure the version is defined in each tutorial and use that instead of global version from `gatsby-node` file. I'll add this to web-board to make sure it's tracked.